cypress/support/commands.ts

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Do we really need this file?

Reply:
Nope! In fact we dont need most of the files in cypress folder now. I was looking at setting up code coverage but it was more trouble than it's worth.